src/styles/card.css

```diff
@@ -0,0 +1,102 @@
+/* code here */
+.product {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+  width: 200px;
+  height: 408px;
+}
+
+.product__photo {
+  margin-bottom: 40px;
+  margin-top: 16px;
+  width: 160px;
+  height: 134px;
+}
+
+.product__description {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.product__code {
+  margin-bottom: 16px;
+  align-self: flex-start;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.product__rating {
+  width: 100%;
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.product__rating-reviews {
+  justify-self: center;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.product__cost {
+  width: 100%;
```

Comment:
again, find next issues like that yourself

Reply:
![image](https://user-images.githubusercontent.com/110053436/216037496-23cb3217-ef1c-4d60-b451-1b23b25a44db.png)

if i change that property - my two blocks move out of distance